### PR TITLE
add snapshot test for the pages directory

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
Snapshot Test:
add three snapshot tests for

- Home page
- Quote page
- Calculator page

### CODE REVIEW

File: src/components/__test__/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+test('test when first press 0', () => {
+  const { getByTestId } = render(<Calculator />);
+  const result = getByTestId('result');
+  const zero = getByTestId('no-zero');
+
+  fireEvent.click(zero);
+
+  expect(result.textContent).toBe('0');
+});
```
Comment (omar-muhamad) [inline src/components/__test__/Calculator.test.js]: - [ ] Please include all `calculate` test cases inside `describe` to group them together like the image below.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @maenkhraisha and @HammadSiddique, 

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add @HammadSiddique name and social links in your README file in the authors section because now this is a collaboration project.
image

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by OLIPLICHE** [comment]
ok

**Review by OLIPLICHE — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**